packages/client/src/components/ui/textarea/textarea.module.scss

```
@@ -0,0 +1,22 @@
+@import '@/assets/variables.scss';
+
+.textarea-wrapper{
+
+.label{
+  display: block;
+  padding:0 0 2px 2px;
+  font-size: 16px;
+  font-weight: 500;
+}
+
+
+.textarea {
+    all: unset;
+    box-sizing: border-box;
+    background-color: $colorMain;
+    display: block;
+    border-radius: 5px;
+    padding: 7px;
+    font-size: 16px;
+  }
+}
```

Рекомендую называть классы по [БЭМ](https://ru.bem.info/methodology/quick-start/#%D0%B1%D0%BB%D0%BE%D0%BA)-у
```suggestion
.textarea-block {
  &__label {
    display: block;
    padding:0 0 2px 2px;
    font-size: 16px;
    font-weight: 500;
  }


  &__input {
    all: unset;
    box-sizing: border-box;
    background-color: $colorMain;
    display: block;
    border-radius: 5px;
    padding: 7px;
    font-size: 16px;
  }
}
```

---

Спасибо, покопаю как совместить  в реакте scss модули и БЭМ

---

БЭМ это методология, т.е. как лучше наименовать css классы чтобы они были очевидными и понятными.